modules/olm-operator-framework.adoc

```diff
@@ -24,7 +24,7 @@ Operator Registry::
 The Operator Registry stores ClusterServiceVersions (CSVs) and Custom Resource
 Definitions (CRDs) for creation in a cluster and stores Operator metadata about
 packages and channels. It runs in a Kubernetes or OpenShift cluster to provide
-this Operator catalog data to the OLM.
+this Operator Catalog data to the OLM.
```

Comment:
Should "Operator Catalog" (capitalized) persist through the other occurrences in this PR (e.g., "a custom Operator Catalog image")?

Reply:
Switching back to "catalog"."